docs/ELRS_RECEIVERS.md

```diff
@@ -54,6 +56,15 @@ Note: In order to send RC channels over the serial connection, change the Rx Snd
     - For receivers with a single (non-RGB) LED you can confirm the receiver is in bootloader mode if the LED is solid.
 - Open a supported browser (Chrome, Edge, Opera, not Firefox) and navigate [here](https://esp.huhn.me/).
 - Click connect and select the serial port of your USB<>UART.
-- Upload the .bin file for your receiver and click program.
+- For ESP8266 based receivers:
+    - Upload the .bin file for your receiver:
+    - <img src="images/ESP8266_Flash.png">
```

Comment:
for some reasons my browser refuses to show it, probably just a problem on my end

note: the "-" does create a dangling o ...  maybe another method of insertuingthe fig would be better, but just nitpick

Reply:
Added in breaks instead, looks cleaner.